setup.py

```diff
@@ -10,7 +10,7 @@
 
 setup(
     name='lucidtech-las',
-    version='3.3.1',
+    version='3.4.0',
```

Comment:
See comments above

Reply:
Maybe the breaking changes should be performed in a bigger refactoring?